api/src/test/java/io/druid/data/input/impl/DelimitedParseSpecTest.java

```diff
@@ -122,8 +129,91 @@ public void testDefaultColumnList()
         ",",
         null,
         null,
+        null,
         false,
         0
     );
   }
+
+  @Test
+  public void testDefaultListDelimiter()
```

Comment:
This test is not properly formatted in the Druid's style

Reply:
fixed

Reply:
It's not fully fixed.

 - In Druid code is never indented by 8 spaces, e. g. see above.
 - Please minimize constructions like
```
new TimestampSpec(
    "timestamp",
    "auto",
    null
)
```
to just `new TimestampSpec("timestamp", "auto", null)`, when it reasonably fits 120 columns by line width.

- Code in lines 213-218 is not properly line-broken

Reply:
fixed